MetovaTestKitTests/TestingUtilities/TestFailureExpectation.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  TestFailureExpectation.swift
```

Comment:
Since there are now separate Basic/Detailed TestFailureExpectation extensions, could put the Basic.. struct in it's own file.